### PR TITLE
Loader when creating survey

### PR DESCRIPTION
**Pull request by ramirobg94**
## Description

- #### Added/Removed/Updated something...
   
- #### Added/Removed/Updated something else...
    

## Types of changes

- [ ] Docs change
- [ ] Refactoring
- [ ] Dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Do you consider this PR needs further testing?

- [ ] No
- [ ] Yes

### CODE REVIEW

File: webapp/components/survey/SurveyCreate/store/actions/useOnCreate.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ export const useOnCreate = ({ newSurvey, setNewSurvey }) => {
 
   return () => {
     ;(async () => {
+      dispatch(LoaderActions.showLoader())
       const data = await sendSurveyCreateRequest({ dispatch, newSurvey, user })
       if (!data) {
```
Comment (SteRiccio) [inline webapp/components/survey/SurveyCreate/store/actions/useOnCreate.js:38]: in this case the loader is never hidden...

Comment (SteRiccio) [reply webapp/components/survey/SurveyCreate/store/actions/useOnCreate.js:38]: maybe you can hide the loader at line 38

**Review by SteRiccio — CHANGES_REQUESTED** [review]

File: webapp/components/survey/SurveyCreate/store/actions/useOnCreate.js (hunk 1 above)
Comment (ramirobg94) [inline webapp/components/survey/SurveyCreate/store/actions/useOnCreate.js:38]:
```suggestion
      dispatch(LoaderActions.hideLoader())
      if (!data) {
```

File: webapp/components/survey/SurveyCreate/store/actions/useOnCreate.js [hunk 2]
```diff
@@ -56,6 +57,7 @@ export const useOnCreate = ({ newSurvey, setNewSurvey }) => {
           validation,
         })
       }
+      dispatch(LoaderActions.hideLoader())
```
Comment (ramirobg94) [inline webapp/components/survey/SurveyCreate/store/actions/useOnCreate.js:60]:
```suggestion
```

**Review by SteRiccio — APPROVED** [review]

**Pull request merged**